.github/workflows/ci.yml

```diff
@@ -20,3 +20,5 @@ jobs:
   puppet:
     name: Puppet
     uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v3
+    with:
+      acceptance_runs_on: ubuntu-22.04
```

Comment:
with the new beaker-docker release, is this still required?

Reply:
No, I dropped it and force-pushed but Github stuck on it 🤔

Reply:
Yay, force-pushed again and it's good now!